packages/keyboard-shortcuts/src/components/shortcut-provider.js

```diff
@@ -20,12 +20,12 @@ const { Provider } = context;
  * @return {import('@wordpress/element').WPElement} Component.
  */
 export function ShortcutProvider( props ) {
-	const keyboardShortcuts = useRef( new Set() );
+	const [ keyboardShortcuts ] = useState( new Set() );
```

Comment:
Noting that `new Set()` will be executed on every rerender, and even though it's not going to be used and it's thrown away immediately, it's one extra thing we can micro-optimize by using [lazy initialization](https://kentcdodds.com/blog/use-state-lazy-initialization-and-function-updates). 
```suggestion
	const [ keyboardShortcuts ] = useState( () => new Set() );
```

Reply:
Good call.